lib/mandrill_dm/message.rb

```diff
@@ -276,6 +276,15 @@ def inline_attachments?
       mail.attachments.any?(&:inline?)
     end
 
+    def html_content?
+      return true if mail.content_type.nil?
+      mail.content_type =~ %r{text/html} ? true : false
```

Comment:
It should be exactly `text/html`, not `anytext/html5` or whatever. Same for `text/plain`.